.github/workflows/unit-integration-test.yml

```diff
@@ -0,0 +1,109 @@
+name: GitHub Actions Workflow
+
+on:
+  pull_request:
+  workflow_dispatch: # Allow manual trigger
+
+jobs:
+  build:
+    runs-on: self-hosted
+
+    env:
+      CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
+      E2E_ENVIRONMENT: 'CI'
+      AB2D_V2_ENABLED: 'true'
+      SEARCH_BENE_BATCH_SIZE: '10'
+      AWS_DEFAULT_REGION: "us-east-1"
+      WORKSPACE: ${{ github.workspace }}
+      AB2D_BFD_KEYSTORE_LOCATION: "${{ github.workspace }}/opt/ab2d/ab2d_bfd_keystore"
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Setup Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: '17'
+          maven-version: '3.6.3'
+
+      - name: Download bfd keystore
+        run: |
+          aws s3 cp s3://bcda-opensbx-access-logs/ab2d_sbx_keystore /tmp/ab2d_sbx_keystore
+
+      - name: Set env vars from AWS params
+        uses: cmsgov/ab2d-bcda-dpc-platform/actions/aws-params-env-action@main
+        env:
+          AWS_REGION: ${{ vars.AWS_REGION }}
+          ECR_REPO_ENV_AWS_ACCOUNT_NUMBER: ${{ vars.SBX_ACCOUNT_NUMBER }}
+        with:
+          params: |
+            ARTIFACTORY_URL=/artifactory/url
+            ARTIFACTORY_USER=/artifactory/user
+            ARTIFACTORY_PASSWORD=/artifactory/password
+            CC_TEST_REPORTER_ID=cc-test-reporter-id
```

Comment:
This should be in aws parameter store under a path like /cc/test-reporter-id, and if it's ab2d-specific it should be in the parameters we grab in the step below.